nginx_proxy/start_nginx.sh

```diff
@@ -31,30 +32,47 @@ configuration is already provided on disk:
 $(command basename $0)
 (2) Starts nginx with a custom Endpoints service name and service version, and
 have nginx obtain the service configuration:
-$(command basename $0) -n ENDPOINTS_SERVICE_NAME -v ENDPOINTS_SERVICE_VERSION
+$(command basename $0) -n ENDPOINTS_SERVICE_NAME \
+-v ENDPOINTS_SERVICE_VERSION -r ROLLOUT_STRATEGY
 Options:
     -h
         Shows this message.
     -n ENDPOINTS_SERVICE_NAME
         Required. The name of the Endpoints Service.
         e.g. my-service.my-project-id.appspot.com
     -v ENDPOINTS_SERVICE_VERSION
-        Required. The version of the Endpoints Service which is assigned
-        when deploying the service API specification.
+        Optional. Required when rollout_strategy is "fixed". Specify the service
+        config to use when ESP starts.  ESP will download the service config
+        with the config id. Forbidden when rollout_strategy is "managed".
         e.g. 2016-04-20R662
+    -r ROLLOUT_STRATEGY
+        Optional. Specify how ESP will update its service config. The value
+        should be either "fixed" or "manage". If it is "fixed", the ESP will
+        keep using the service config when it starts.  If it is "managed",
+        ESP will constantly check the latest rollout, use the service configs
+        specified in the latest rollout. If it is not specified, "fixed"
+        would be chosen by default.
+        e.g. fixed
     -
 END_USAGE
   exit 1
 }
-while getopts 'ha:n:N:p:S:s:v:' arg; do
+while getopts 'ha:n:N:p:S:s:v:r:' arg; do
   case ${arg} in
     h) usage;;
     n) ENDPOINTS_SERVICE_NAME="${OPTARG}";;
     v) ENDPOINTS_SERVICE_VERSION="${OPTARG}";;
+    r) ENDPOINTS_ROLLOUT_STRATEGY="${OPTARG}";;
     ?) usage;;
   esac
 done
 
+if [[ "${ENDPOINTS_ROLLOUT_STRATEGY}" == "managed" && \
```

Comment:
Add a check for when rollout strategy is fixed and no version is provided. This should give an error as well.